mode/examples/Topics/File IO/SaveFrames/SaveFrames.pyde

```diff
@@ -0,0 +1,61 @@
+"""
+Save Frames
+by Daniel Shiffman.
+
+This example demonstrates how to use saveFrame() to render
+out an image sequence that you can assemble into a movie
+using the MovieMaker tool.
+"""
+
+# A boolean to track whether we are recording are not
+recording = False
+
+
+def setup():
+    size(640, 360)
+
+
+def draw():
+    background(0)
+
+    # An arbitrary oscillating rotating animation
+    # so that we have something to render
+    for a in xrange(0, 6, 1):
```

Comment:
This is not how Daniel Shiffman's example works; goes between 0 and 2pi by 0.2 at a time.

I don't have time to evaluate every line of code in this pull request, and I don't think there's anything to be gained from further work. I'm sorry you've spent so much time on it, but I'm closing this request.